assets/js/src/core/components/grid/columns/types/textarea/textarea-cell.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type CellContext } from '@tanstack/react-table'
+import React, { useState } from 'react'
+import { useEditMode } from '../../../edit-mode/use-edit-mode'
+import { useStyle } from './textarea-cell.styles'
+import TextArea from 'antd/es/input/TextArea'
+import { respectLineBreak } from '@Pimcore/utils/helpers'
+
+export type DefaultCellProps = CellContext<any, any>
+
+export const TextareaCell = (props: DefaultCellProps): React.JSX.Element => {
+  const { isInEditMode, disableEditMode, fireOnUpdateCellDataEvent } = useEditMode(props)
+  const { styles } = useStyle()
+  const [textAreaValue, setTextAreaValue] = useState(String(props.getValue() ?? ''))
+
+  // useEffect(() => {
```

Comment:
I think we can remove that comment? :) 

But now, where I see the useEffect. Do we focus the textarea, the moment we enter the editmode?
Otherwise I would prefer that we still add this. It's saves a click. :D